src/android/toga_android/widgets/base.py

```diff
@@ -131,5 +131,5 @@ def align(value):
         LEFT: Gravity.LEFT,
         RIGHT: Gravity.RIGHT,
         CENTER: Gravity.CENTER_HORIZONTAL,
-        JUSTIFY: Gravity.CENTER_HORIZONTAL,
+        JUSTIFY: Gravity.LEFT,
```

Comment:
I think making JUSTIFY left-aligned is a more obvious choice than centered, and both Windows and GTK already do that. 

In the current state, each of the lines in a newline-separated string will be treated as the last line in a paragraph, so they wouldn't be justified anyway. JUSTIFY will therefore behave the same as LEFT for the moment. 

Actual justification will have to wait until we have word wrap. We could then consider adding something like CSS's [`text-align-last`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align-last) to control the alignment of the last line in a paragraph.